README.md

```diff
@@ -1,2 +1,9 @@
 # huffman-image-compression
-Huffman Image Compression
+Huffman Image Compression is one of the basic compression methods, that have proven useful in image and video compression standards. When applying Huffman encoding technique on an Image, the source symbols can be either pixel intensities of the Image, or the output of an intensity mapping function.
+
+# Usage
```

Comment:
This looks like it should be within three backticks ```

Reply:
Fixed